### PR TITLE
Update offline blog to match v0.23.2

### PR DESCRIPTION
**Pull request by Sunyang730**
- Add the correct description for generating offline-mirror
yarn.lock file with v0.23.2
- Add tips for running yarn install with offline flag

### CODE REVIEW

**Comment by Sunyang730** [comment]
@bestander Here are the changes.

File: _posts/2016-11-24-offline-mirror.md [hunk 1]
```diff
@@ -105,33 +116,32 @@ success Saved lockfile.
...
   version "2.1.13"
-  resolved mime-types-2.1.13.tgz#e07aaa9c6c6b9a7ca3012c69003ad25a39e92a88
+  resolved "https://registry.yarnpkg.com/mime-types/-/mime-types-2.1.13.tgz#e07aaa9c6c6b9a7ca3012c69003ad25a39e92a88"
   dependencies:
     mime-db "~1.25.0"
 ```
 
-The `resolved` field for each resolution should now contain file names relative to the `npm-packages-offline-cache`  folder that was configured earlier.
-Each `resolved` dependency also contains a checksum after the file name to ensure that no one mangles with the downloaded files.
+The offline cache filename is derived from the base name of the resolved URL. It will be store in the npm-packages-offline-cache folder that was configured earlier. Each resolved dependency also contains a checksum after the file name to ensure that no one mangles with the downloaded files.
```
Comment (bestander) [inline _posts/2016-11-24-offline-mirror.md]: It will be stored

**Comment by bestander** [comment]
Big thank you for updating the post!
Just one type fix and ready to merge

**Comment by Sunyang730** [comment]
Done

**Comment by Sunyang730** [comment]
hum, curious, when will it be available? @bestander

**Comment by bestander** [comment]
It is available in 0.23.2

**Pull request merged**